mesonbuild/cargo/interpreter.py

```diff
@@ -68,6 +68,13 @@ class PackageConfiguration:
     required_deps: T.Set[str] = dataclasses.field(default_factory=set)
     optional_deps_features: T.Dict[str, T.Set[str]] = dataclasses.field(default_factory=lambda: collections.defaultdict(set))
 
+    def get_features_args(self) -> T.List[str]:
+        """Get feature configuration arguments."""
+        args = []
```

Comment:
`args: T.List[str]  = []`

Reply:
It should, but doesn't. It sees that and automatically assigns it `list[Any]`. at least, that's what mypy through vscode provides in those situations.